common/reviews/api/node-library-build.api.ts

```diff
@@ -1,13 +1,9 @@
 export declare function addSuppression(str: string): void;
 
 class ApiExtractorTask extends GulpTask<IApiExtractorTaskConfig> {
-  // (undocumented)
```

Comment:
@pgonzal @nickpape-msft so is there any action item on my part here to unblock the PR and get this in?